src/idiom248.fuzion

```diff
@@ -0,0 +1 @@
+d := (if s then -1.0 else 1.0) * m * (2 ** e).as_f64
```

Comment:
This will not work for `e>32`, so maybe try this
```suggestion
d := (if s then -1.0 else 1.0) * m * (2.0 ** e)
```
Also check if there is a function missing in our base lib that is useful in other languages. If so, maybe add an issue to the Fuzion repository and reference it in this code.